doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-groups-ref.adoc

```diff
@@ -0,0 +1,87 @@
+[id='security-management-rest-api-groups-ref_{context}']
+= Groups
+
+The Security Management REST API supports the following endpoints for managing groups in {CENTRAL}. The Security Management REST API base URL is `http://SERVER:PORT/business-central/rest/`. All requests require HTTP Basic authentication or token-based authentication for the `admin` user role.
```

Comment:
Search all instances of `business-central/rest` and replace with `{URL_COMPONENT_CENTRAL}/rest`, but thoughtfully of course. This shouldn't be hard-coded because in DM, it's still `decision-central/rest`, as you can see elsewhere in that Knowledge Store chapter in the DM version. But do check with your SME to make sure this is still the case. If there has been a change and it is in fact buiness-central in DM, then a larger update is in order. Last I knew (a few months ago), there was still the division for backward compatibility.

Also, add a back slash to the URL to kill the link, as I've done in the previous sections and elsewhere: `\http://SERVER:PORT/...`  The slash is not rendered in the output, but the link is disabled. I recommend doing that in all cases, especially when it's a template. Can be an actual link only when you truly want the user to click it and go there.